ygot/render.go

```diff
@@ -50,6 +50,9 @@ type gnmiPath struct {
 	stringSlicePath []string
 	// pathElemPath stores a path expressed as a series of PathElem messages.
 	pathElemPath []*gnmipb.PathElem
+	// isAbsolute determines whether the stored path is absolute (when set), or relative
+	// when unset.
+	isAbsolute bool
```

Comment:
Now that this is a full class you could get the same functionality by having a nil first element and isAbsolute() method.

Reply:
I didn't take this approach because specifically the encoding `[]string{""}` does not mean `/` in the gNMI path encoding. I decided an explicit signal was better such that the caller could indicate that it was absolute vs. not.